spec/models/miq_web_server_worker_spec.rb

```diff
@@ -0,0 +1,8 @@
+RSpec.describe MiqWebServiceWorker do
+  it "preload_for_worker_role autoloads api collection classes and descendants" do
+    allow(EvmDatabase).to receive(:seeded_primordially?).and_return(true)
+    expect(MiqWebServiceWorker).to receive(:configure_secret_token)
+    MiqWebServiceWorker.preload_for_worker_role
+    expect(defined?(ServiceAnsibleTower)).to be_truthy
```

Comment:
This is a "meh" test.  I can verify it when running the test independently, but as part of the test suite, it's possible this constant is already loaded by the time this test is run.

Reply:
it will be a sporadic failure at least

Reply:
Well, thankfully, it will just pass without verifying the actual code loads it  properly before/after.